build.sbt

```diff
@@ -95,6 +95,7 @@ lazy val releaseSettings = {
       tagRelease,
       // For non cross-build projects, use releaseStepCommand("publishSigned")
       releaseStepCommandAndRemaining("+publishSigned"),
+      releaseStepCommand("docs/ghpagesPushSite"),
       setNextVersion,
       commitNextVersion,
       releaseStepCommand("sonatypeBundleRelease"),
```

Comment:
Is this step necessary? Snapshot publishing *should* be done via CI, isn't it?

Reply:
This is how we publish releases to Sonatype, not snapshots, so yes.

I've been running `sbt release` locally.  Ideally, we want to do that step from CI as well.  That's under discussion on #28, which I need to go reread myself. :smile: